### DISCUSSION TITLE
Account blocked, whilst registrating. Please unblock my GitHub account.

### DISCUSSION THREAD

**Discussion by ThomPoppins** [vercel/community#3043] [Help]
### Description

Please unblock this GitHub account. I will use your deployment services for testing/study purposes, only I was already blocked before I could login even once. 

This is my single GitHub account, so I need this account to be unblocked please, I'll always behave and never misuse your services.

I tried to register with my email address but that wasn't possible either, resulted in an error. 

> We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup)
for further details

Please help me, thank you in advance!

**Comment by amyegan** [comment]
Hi @ThomPoppins. Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team for help with registration 🙏
